src/core/Form/InputToggleButton/InputToggleButton.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { Component, RefObject } from 'react';
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import { HtmlButtonProps, HtmlButton } from '../../../reset';
+import { SuomifiThemeProp, SuomifiThemeConsumer } from '../../theme';
+import { VisuallyHidden } from '../../VisuallyHidden/VisuallyHidden';
+import { Icon } from '../../Icon/Icon';
+import { baseStyles } from './InputToggleButton.baseStyles';
+
+const baseClassName = 'fi-input-toggle-button';
+
+const InputToggleButtonClassNames = {
+  icon: `${baseClassName}_icon`,
+};
+
+export interface InputToggleButtonProps extends HtmlButtonProps {
+  open: boolean;
+  buttonRef?: RefObject<HTMLButtonElement>;
```

Comment:
This is internal component like FilterInput is, but should we add some JSDoc for these props?

Reply:
Added for the necessary parts.